### ISSUE TITLE
proper backup of non-resudential keys?

### ISSUE THREAD

**Issue by gurkburk76** [NoMoreFood/putty-cac#129]
So i'm testign a bit with the residential and non residential keys and a yubi key 5.
I'm looking at the registry and the private key seems to be stored here:

Dator\HKEY_USERS\S-1-5-21-30xxx3-153xxxxx69-3xxxx18672-xxxx\SOFTWARE\SimonTatham\PuTTY\Fido\
masked the SID a bit, but is this correct? Would be nifty if there was an import/export button(s) in the future :)

**Comment by NoMoreFood** [comment]
Yes, for non-resident keys that is what you will need to export share the protected keys between accounts.  For resident keys, the import tool can do those.  I'll consider adding something in the future.
